.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: PR - Lint
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+        cache: 'npm'
+    - name: Install modules
+      run: npm ci
+    - name: Run ESLint (TypeScript)
+      run: npm run lint:fix
```

Comment:
Should this not be `npm run lint` `fixing` lint changes in the pipeline will not be written to the repo.

Reply:
that's true. I was testing it to see if it changes is in the pipeline - but it doesn't. So I'm taking it out